map/src/manager/track/SaveTrackManager.js

```diff
@@ -166,6 +155,43 @@ export function saveTrackToLocalStorage({ ctx, track }) {
     });
 }
 
+export async function saveEmptyTrack(folderName, ctx) {
+    //create empty file
+    const convertedData = new TextEncoder().encode('');
+    const zippedResult = require('pako').gzip(convertedData, { to: 'Uint8Array' });
+    const convertedZipped = zippedResult.buffer;
+    const oMyBlob = new Blob([convertedZipped], { type: 'gpx' });
+
+    //create data
+    const data = new FormData();
+    data.append('file', oMyBlob, 'empty');
+    const params = {
+        type: 'GPX',
+        name: folderName + '/empty' + '.ignore',
+    };
+    //save empty file with new folder's name
+    const res = await apiPost(`${process.env.REACT_APP_USER_API_SITE}/mapapi/upload-file`, data, { params });
+    if (res && res?.data?.status === 'ok') {
+        refreshGlobalFiles(ctx, params).then();
+        return true;
+    }
+}
+
+async function refreshGlobalFiles(ctx, params) {
```

Comment:
Slightly unclear what are `params` and why only 1 of of params is used. Better to rename params to smth such as `currentFile` or `currentFileName` and additionally change caller's `params` to `params?.name`.